platform/src/main/java/org/stellar/anchor/platform/data/JdbcSep24Transaction.java

```diff
@@ -97,34 +104,44 @@ public void setRefunds(Sep24Refunds refunds) {
    * bank account.
    */
   @SerializedName("to_account")
+  @Column(name = "to_account")
   String toAccount;
 
   @SerializedName("request_asset_code")
+  @Column(name = "request_asset_code")
   String requestAssetCode;
 
   @SerializedName("request_asset_issuer")
+  @Column(name = "request_asset_issuer")
   String requestAssetIssuer;
 
   /** The SEP10 account used for authentication. */
-  @SerializedName("sep10_account")
+  @SerializedName("sep10account")
+  @Column(name = "sep10account")
   String sep10Account;
```

Comment:
The serialized name still needs to be `sep10_account` right? Same for the serialized name of the SEP-10 memo?

Reply:
Unfortunately, Spring maps to `sep10Account` to `sep10account` and `sep10AccountMemo` to `sep10account_memo`. That's why the JPA Buddy has this strange bug. 

I am locking the name so that JPA Buddy will be consistent with Spring.

Reply:
I think `SerializedName` shouldn't be changed

Reply:
I agree, unless `SerializedName` needs to match `Column`, I would keep `SerializedName` unchanged.